packages/itwin/ec3-widget/src/components/HorizontalTile.scss

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import "~@itwin/itwinui-css/scss/variables";
+
+.rcw-horizontal-tile-container {

```

Comment:
Should be `.ec3-horizontal-tile-container`

Reply:
Selector renamed